packages/visitor-keys/src/index.ts

```diff
@@ -1,2 +1,3 @@
 export { getKeys } from './get-keys';
-export { visitorKeys, VisitorKeys } from './visitor-keys';
+export { visitorKeys } from './visitor-keys';
+export type { VisitorKeys } from './visitor-keys';
```

Comment:
we should probably turn on our `consistent-type-exports` rule

Reply:
Yeah all of these `type` changes allowed compilation `isolatedModules` to work